main.js

```diff
@@ -60,7 +60,18 @@ startTimerButton.addEventListener('click', function() {
 logActivityButton.addEventListener('click', logActivityEvents);
 createNewActivityButton.addEventListener('click', returnToActivityForm);
 
-//Add functions below
+// Add functions below
+(function populatePastActivities() {
+  var keys = Object.keys(localStorage);
```

Comment:
**Question:** Have y'all learned about `let` and `const`?
If so, use those here. If not, ignore this comment.

Reply:
Hello - It's been introduced but more for awareness and future knowledge but not as a requirement yet.